dynamo/write.js

```diff
@@ -15,26 +15,26 @@ const docClient = new AWS.DynamoDB.DocumentClient();
  * Adds or updates DB with new Gmail stats
  * @param {Stats} input Gmail stats object
  */
-const write = function (input) {
+const write = async function (input) {
+  if (!input.hasOwnProperty("time"))
+    throw "gmail-stats::save::inputError - no 'time' attribute";
```

Comment:
Replace throw with return

Reply:
Is it preferred to throw an error object so that the program stops executing during runtime?

Reply:
Yeah, you would want to stop the execution. The code is typically structured in a way if it fails, it only fails for that one job (e.g. fetching email for one person) so it's okay for us to throw.

Reply:
In this case, we can swap it out with new Error('No time attribute')